cookbooks/aws-parallelcluster-slurm/resources/munge/munge_centos7.rb

```diff
@@ -0,0 +1,20 @@
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+provides :munge, platform: 'centos' do |node|
+  node['platform_version'].to_i == 7
+end
+
+use 'partial/_munge_actions'
+use 'partial/_munge_rhel'
+
+def munge_libdir
```

Comment:
minor: you can avoid to define this since it's already defined in the `_munge_rhel`